loopout.go

```diff
@@ -86,12 +86,25 @@ func newLoopOutSwap(globalCtx context.Context, cfg *swapConfig,
 		return nil, fmt.Errorf("cannot initiate swap: %v", err)
 	}
 
+	swapInvoiceHash, err := swap.GetInvoiceHash(
+		cfg.lnd.ChainParams, swapResp.swapInvoice,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	if swapInvoiceHash != swapHash {
+		return nil, fmt.Errorf(
+			"cannot initiate swap, swap invoice hash %s not equal generated swap hash %s",
+			swapInvoiceHash.String(), swapHash.String())
```

Comment:
If you use `%v`, the `String()` call is not necessary.

Reply:
Fixed!